alot/db/manager.py

```diff
@@ -385,6 +385,9 @@ def get_threads(self, querystring, sort='newest_first'):
         assert sort in self._sort_orders
         q = self.query(querystring)
         q.set_sort(self._sort_orders[sort])
+        if exclude_tags:
```

Comment:
Eh, I don't really see the point when you're not going to modify the argument, and I personally dislike using a type other than the expected type as the default one more, since that looks rather misleading.